utilities/pipelines/resourceRemoval/helper/Invoke-ResourceRemoval.ps1

```diff
@@ -154,13 +154,14 @@ function Invoke-ResourceRemoval {
             $subscriptionId = $resourceId.Split('/')[2]
             $resourceGroupName = $resourceId.Split('/')[4]
             $resourceName = Split-Path $resourceId -Leaf
-            $purgePath = 'subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.MachineLearningServices/workspaces/{2}?api-version=2023-04-01-preview&forceToPurge={3}' -f $subscriptionId, $resourceGroupName, $resourceName, $true
+            $purgePath = '/subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.MachineLearningServices/workspaces/{2}?api-version=2023-04-01-preview&forceToPurge={3}' -f $subscriptionId, $resourceGroupName, $resourceName, $true
             $purgeRequestInputObject = @{
                 Method = 'DELETE'
                 Path   = $purgePath
             }
             if ($PSCmdlet.ShouldProcess("Machine Learning Workspace [$resourceName]", 'Remove')) {
-                $null = Invoke-AzRestMethod @purgeRequestInputObject
+                #$null = Invoke-AzRestMethod @purgeRequestInputObject
```

Comment:
```suggestion
```

Reply:
Currently stuff I am testing so will change when I put the PR in ready.